ssize_t used in gdft.c won't compile under MS compiler

Trying to build libgd for conda-forge on Windows (https://github.com/conda-forge/libgd-feedstock/pull/39), we are getting:

```
2021-03-06T20:55:56.6159297Z 	cl /nologo /O2 /Qvec-report:1 /MD /TC  /DWIN32=1 /D_WIN32=1 /DMSWIN32=1 /DBGDWIN32=1 /DENABLE_GD_FORMATS=1 /DHAVE_FT2BUILD_H=1 /DHAVE_GD_H=1 /DHAVE_ICONV_H=1 /DHAVE_LIBFREETYPE=1 /DHAVE_LIBJPEG=1 /DHAVE_LIBPNG=1 /DHAVE_LIBWEBP=1 /DHAVE_LIBZ=1 /DHAVE_LIBXPM=1 /DHAVE_LIBTIFF=1 /Isrc /I..\_build_env\Library\bin\include -I..\_build_env\Library\bin\include\libpng16 -I..\_build_env\Library\bin\include\freetype -I..\_build_env\Library\bin\include\freetype2 /Itests /Itests\gdtest -I..\_build_env\Library\bin\include /c /Fo".\gdbuild\gdft.obj" /Fd.\gdbuild\gdft.pdb src\gdft.c
2021-03-06T20:55:56.6310441Z gdft.c
2021-03-06T20:55:57.1072856Z src\gdft.c(445): error C2061: syntax error: identifier 'textLayout'
2021-03-06T20:55:57.1200163Z src\gdft.c(445): error C2059: syntax error: ';'
2021-03-06T20:55:57.1201686Z src\gdft.c(445): error C2059: syntax error: '<parameter-list>'
2021-03-06T20:55:57.1202923Z src\gdft.c(1115): error C2065: 'ssize_t': undeclared identifier
2021-03-06T20:55:57.1204662Z src\gdft.c(1115): error C2146: syntax error: missing ';' before identifier 'count'
2021-03-06T20:55:57.1228980Z src\gdft.c(1115): error C2065: 'count': undeclared identifier
2021-03-06T20:55:57.1230574Z src\gdft.c(1414): error C2065: 'count': undeclared identifier
2021-03-06T20:55:57.1232193Z src\gdft.c(1416): error C2065: 'count': undeclared identifier
2021-03-06T20:55:57.1235141Z src\gdft.c(1424): error C2065: 'count': undeclared identifier
2021-03-06T20:55:57.1426077Z NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\cl.EXE"' : return code '0x2'
```
Despite it being called a syntax error, I think it's actually this problem:

`2021-03-06T20:55:57.1202923Z src\gdft.c(1115): error C2065: 'ssize_t': undeclared identifier`

Anyway, according to this issue on some totally unrelated project:

https://github.com/vlm/asn1c/issues/159

ssize_t is a posix extension. That issue suggests this as a fix:

```
#if defined(_MSC_VER)
#include <BaseTsd.h>
typedef SSIZE_T ssize_t;
#endif
```
However, I searched the code and this is the only place `ssize_t` is used. 

Does that need to be a signed result? Or would size_t work just as well?

Note: we first had this issue with 3.2.1 -- I was hoping someone else would find it and fix it, when it cropped up again with 3.2.2, I figured I'd dig more to figure it out. This seems very easy to fix.

Thanks.

--- a/windows/Makefile.vc
+++ b/windows/Makefile.vc
@@ -146,6 +146,7 @@ CFLAGS= $(CFLAGS) \
        /DHAVE_LIBZ=1\
        /DHAVE_LIBXPM=1\
        /DHAVE_LIBTIFF=1\
+       /DHAVE_CONFIG_H\
        $(INCLUDES)
 
 TEST_CFLAGS=$(CFLAGS)
@@ -225,6 +226,8 @@ make_dirs:
 	@echo #ifndef HAVE_LIBTIFF>> $(GD_CONFIG_H)
 	@echo #define HAVE_LIBTIFF>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
+	@echo #define ssize_t SSIZE_T>> $(GD_CONFIG_H)
+	@echo #define SSIZE_MAX MAXSSIZE_T>> $(GD_CONFIG_H)
 
 run_tests: pretest
 	@cd $(TDR)

